### DISCUSSION TITLE
I have a problem creating a new profile repository. I get the message: ‘The repository [Repository Name] has been retired and cannot be reused.

### DISCUSSION THREAD

**Discussion by security-engineer** [community/community#136347] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I renamed my GitHub username, so I tried to rename my config special repository. However, I received an error message: ‘The repository [Repository Name] has been retired and cannot be reused.’ How can I solve this problem? From what I’ve found, it seems that only an admin can resolve this issue.

**Comment by mhaemnn** [comment]
It is possible that the previous repository must be deleted first

**Reply by security-engineer** [reply]
how can i delete previous repository? I have not made a repository that i want

**Reply by mhaemnn** [reply]
Sorry, after I researched it again, it looks like the case you are experiencing needs to be helped directly by the GitHub team. The following is the same case as you experienced

https://stackoverflow.com/questions/62298084/the-repository-name-has-been-retired-and-cannot-be-reused

**Reply by security-engineer** [reply]
I tried to change my previous username and made a current username repository. and then tried to back to my current username. I received an error message: Username change was not successful. The repository name solarvim/solarvim has been retired and cannot be reused. so i need to helped by Github Team... TT

**Reply by mhaemnn** [reply]
That's right, you need help from the Gitub team, has anyone read the link I provided before?

**Comment by AjmalDevala** [comment]
Response #1
